chirpr.py

```diff
@@ -119,9 +130,35 @@ def chirps():
     return render_template('admin/chirps.html', chirps=chirp_list)
     
     
+@app.route('/chirps/<handle>')
+def chirps_hn(handle):
+    uid = db_access.get_id(handle)
+    chirp_list = db_access.get_chirps(db_access.get_id(handle))
+    return render_template('chirp.html', chirps=chirp_list, chirp_from_page=False, markdown=markdown2.markdown, name=db_access.get_user(session['user']))
+
+    
+@app.route('/chirp', methods=['GET', 'POST'])
+def chirp():
+    if 'user' in session:
+        user = session['user']
+        MD = False
+        if request.method == 'POST':
+            content = request.form["content"]
+            if len(content) in range(1,360) and content != '':
```

Comment:
This is redundant. If `len(content) > 0`, then `content != ''`

Also, `1 <= len(content) <= 360` is clearer.